Updated docs for recent command. (#471)
hugovk/pypistats: README.md

                        [-v]
                        package

> Retrieve the aggregate download quantities for the last 1/7/30 days, only
> includes without mirror downloads

Reviewer: ```suggestion
Retrieve the aggregate download quantities for the last 1/7/30 days,
excluding downloads from mirrors
```
